### DISCUSSION TITLE
Multiple voices

### DISCUSSION THREAD

**Discussion by GilbertGomez94** [openai/whisper#236] [Q&A]
If I have an audio file with multiple voices from a voice call, should whisper be available to transcribe the conversation? I'm trying to test it, but I only get the transcript of one speaker, not both.

**Comment by carlosalosno** [comment]
Hi there

you will take a look on that project, can do speaker diarization 

https://github.com/mave5/podalize

**Comment by alhafoudh** [comment]
Try this https://huggingface.co/pyannote

**Reply by Tortoise17** [reply]
Thank you, do you have any idea how can we take two transcripts overlapping at same time using the whisper while pyannote gives only times for speakers. If you can hint or guide would be great help.

**Comment by Majdoddin** [comment]
Hi @GilbertGomez94, 
[www.lexicaps.com](http://www.lexicaps.com/) seamlessly adds diarization to Whispers transcription. No 3rd party packages.
Feel free to contact me if you need more features.
Announcement: https://github.com/openai/whisper/discussions/1537
Repo: https://github.com/Majdoddin/lexicaps
